src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import {StyledHeading2, StyledHeading4, StyledHeading6} from '../../../ignitus-UserInterfaceBook/styles';
+import * as P from '../styles';
+import {AppIcon} from '../../../ignitus-Shared/types/iconsTypes/iconEnums';
+import {
+  ButtonBottomRight,
+  RoundedButton,
+} from '../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/buttons';
+import {AboutContent} from '../styles';
+
+
+const ProfileDetailView = () => (
+  <P.Container1>
```

Comment:
Please don't name styles with suffix 1,2,3.
You can name this one only Container is the top most one.

Reply:
okay, I'll edit that.